apps/prairielearn/src/lib/editors.ts

```diff
@@ -973,6 +991,46 @@ export class CourseInstanceCopyEditor extends Editor {
   }
 }
 
+async function updateInfoAssessmentFilesForInstanceCopy(
+  courseInstancePath: string,
+  fromCourseSharingName: string,
+) {
+  const assessmentsPath = path.join(courseInstancePath, 'assessments');
+  const assessmentDirs = await fs.readdir(assessmentsPath);
```

Comment:
This doesn't appear to handle nested assessment directories, e.g. `assessments/homeworks/hw1/infoAssessment.json`. It'll need to do that.

Are there tests for this? If so, add that to the tests first to confirm that the fix works. If not, consider adding some!

Reply:
done in 4d8e600e0e